pkg/metrics/metrics.go

```diff
@@ -245,21 +246,29 @@ func (c *Config) Configure(ctx context.Context) error {
 	return nil
 }
 
-// validate will verify if cluster monitoring is enabled in the operator namespace.
-// If the label is not present, it will log and send warning events to the user.
+// validate will verify if cluster monitoring is enabled in the operator namespace. If the label is set false or not
+// present, it will log and send warning events to the user. If the label hold a non-boolean value, returns error.
 func (c *Config) validate(ctx context.Context) (bool, error) {
 	// validate if metrics label is added to namespace
 	wmcoNamespace, err := c.CoreV1().Namespaces().Get(ctx, c.namespace, metav1.GetOptions{})
 	if err != nil {
 		return false, errors.Wrap(err, "error getting operator namespace")
 	}
-	if wmcoNamespace.Labels["openshift.io/cluster-monitoring"] != "true" {
-		metricsEnabled = false
+
+	boolVal := false
```

Comment:
Instead of bool val, can we give it a monitoring specific name? maybe monitoringEnabled?

Reply:
I opted for the non-specific name because there's already a global variable by the name of `metricsEnabled` (which gets assigned to the `boolVal` later in this function). I can make this `labelValue` instead?